cmd/river/go.mod

```diff
@@ -0,0 +1,26 @@
+module github.com/riverqueue/river/cmd/river
+
+go 1.21.4
+
+replace github.com/riverqueue/river => ../..
+
+replace github.com/riverqueue/river/riverdriver/riverpgxv5 => ../../riverdriver/riverpgxv5
```

Comment:
So the whole point of this is that `cmd/river` will not use a `replace` in the end, but I found that I had to have one for now because you can't build this module without one given the current release contains `cmd/river` as a subpackage:

```
$ go test .
ambiguous import: found package github.com/riverqueue/river/cmd/river in multiple modules:
        github.com/riverqueue/river v0.0.12 (/Users/brandur/Documents/go/pkg/mod/github.com/riverqueue/river@v0.0.12/cmd/river)
        github.com/riverqueue/river/cmd/river (/Users/brandur/Documents/projects/river/cmd/river)
```

So I believe what we could do is (1) merge this, (2) cut a new release, and (3) update this package to use that new release and remove this `replace`, then (4) uncomment the other `replace`s in the project.